proposals/1/loadSet.md

```diff
@@ -0,0 +1,129 @@
+# Load Set
+
+## Current situation
+
+-  The hardcoded path for mod installation or discovery is `assets/mods/`. The mod must exist there to be found. 
+- Big mods that may conflict have to be individually disabled to minimize issues.
+- It is harder to find source of problem with big mods potentially messing with smaller mods.
+
+- Downgrading is impossible if one mod requires a version different from another.
+
+## Solution
+
+Allow users to create a set of mods referenceable by a name.
+
+### Advantages
+
+- One set can have a different version of the same mod for another set.
+
+- Decreases complexity of debugging if used.
+
+- No longer necessary to disable mods that might conflict with another mod.
+
+
+### Disadvantages
+
+- Increased implementation complexity.
+
+- Increase storage usage.
+
+
+
+## Solution 1
```

Comment:
`Solution 1` seems to be a sub of `Solution` which doesn't make sense. Maybe rename it to something like `Option 1` ,`Option A` or `Implementation`